sort_resource_links.py

```diff
@@ -0,0 +1,149 @@
+"""
+    If you added a new resource to an existing resources.md or learning-material.md file, just run this script directly.
+    If you added a new resources.md or learning-material.md file, add the path to the file to the RESOURCE_FILE_PATHS list
+    below and then run this script.
+
+    To run this script, open a terminal or command prompt and navigate to the directory that contains this script.
+
+    If you're using a Mac or a Linux distribution, you can run this script from the terminal by typing the following command:
+        python3 sort_resource_links.py
+    If you're using Windows, you can run this script from the command prompt by typing the following command:
+        python sort_resource_links.py
+
+    Note 1: You must have Python 3 installed on your computer to run this script. If you don't have Python 3 installed,
+    you can download it from https://www.python.org/downloads/. Make sure you download the version for your operating
+    system (Windows, Mac, or Linux).
+
+    Note 2: This script assumes that it is at the root of the repository. If you move this script to a different location,
+    you will need to update the RESOURCE_FILE_PATHS list below to include the correct paths to the resources.md and
+    learning-material.md files.
```

Comment:
Hello @milkiyd, we should write a 'guide' to explain this instead of let it here in the `.py` code